templates/partners/channel-and-reseller.html

```diff
@@ -7,35 +7,45 @@
-        <a href="https://partner-portal.canonical.com" class="p-button"><span>Login to the partner portal</span></a>
-      </div>
-      <div class="col-4 u-hide--small u-hide--medium u-align--center u-vertically-center">
-        <img src="https://assets.ubuntu.com/v1/80472358-Canonical-reseller-icon.svg" alt="">
-      </div>
+<section class="p-strip u-no-padding--bottom">
+  <div class="row">
+    <div class="col-8">
+      <h1>Channel&sol;Reseller Programme</h1>
+      <p>Canonical works with channel and reseller partners around the world to ensure that Canonical offerings are available for their customers. Additionally these partners can both resell services and value-added offerings around our technology enabling them to grow their business.</p>
+      <p>Canonical is an industry leader in OpenStack private clouds, the reference platform for all major public clouds, and the open source software operating system that runs from the desktop and all internet connected things. Our success helps our partners build a business around our technology.</p>
+      <a href="https://partner-portal.canonical.com" class="p-button--positive"><span>Login to the partner portal</span></a>
+    </div>
+    <div class="col-4 u-hide--small u-hide--medium u-align--center u-vertically-center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/44842949-canonical-reseller-icon-grey.png",
+        alt="",
+        width="859",
+        height="824",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
     </div>
   </div>
 </section>
 
-<section class="p-strip--light is-shallow" style="margin-top: -1px;">
+<section class="p-strip u-no-padding--top">
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
   <div class="row">
-    <div class="col-8 col-medium-4">
-      <p>Interested in becoming an Ubuntu partner? Talk to us today to learn more about our programmes.</p>
-    </div>
-    <div class="col-4 col-medium-2">
-      <a href="/partners/become-a-partner" class="p-button--positive">Become a partner</a>
+    <div class="col-6">
+      <p class="p-heading--tag">Partners</p>
+      <p class="p-heading--4">Interested in becoming an Ubuntu partner? Talk to us <br> today to learn more about our programmes.</p>
+      <p><a href="" class="p-button--positive">Become a partner</a></p>
     </div>
   </div>
+  </div>
 </section>
 
+
 {% if partners %}
-<section class="p-strip is-deep">
+<section class="p-strip has-accent-border--top-left is-deep">
   <div class="u-fixed-width p-logo-section">
     <h4 class="p-logo-section__title p-muted-heading u-no-max-width u-align--center">Meet some of our channel/reseller partners</h4>
```

Comment:
"Meet some of our channel partners" on doc. The logos are different on the doc as well